scripts/gceworker.sh

```diff
@@ -54,6 +55,22 @@ case ${1-} in
     shift
     gcloud compute ssh "${name}" "$@"
     ;;
+    mount)
+    shift
+    if [ $# != 2 ]; then
+      echo "usage: $0 <remote-path> <local-path>"
```

Comment:
Eh, I think I prefer explicitly choosing what goes where e.g. defaulting to just dropping cockroach into GOPATH, without vendoring, could let the remote glockfile, at a different rev, run amuck of my local GOPATH.